<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralComputerTab</class>
 <widget class="QWidget" name="GeneralComputerTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="name_label">
     <property name="text">
      <string notr="true">PLACEHOLDER</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="logonNamePreWindows2000Label">
       <property name="text">
        <string>Logon name (pre-Windows 2000):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="dNSHostNameLabel">
       <property name="text">
        <string>DNS Host Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="dns_host_name_edit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="description_edit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="locationLabel">
       <property name="text">
        <string>Location:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="location_edit"/>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="sam_name_domain_edit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="sam_name_edit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
